try.go

```diff
@@ -173,7 +173,10 @@ func HandleF(errptr *error, fn func()) {
 // The wrapping includes the file and line of the runtime frame in which it occurred.
 // F pairs well with testing.TB.Fatal and log.Fatal.
 func F(fn func(...any)) {
-	r(recover(), func(w wrapError) { fn(w) })
+	r(recover(), func(w wrapError) {
+//line try.go:1
```

Comment:
Note that this impacts every single line thereafter. That means that uncaught errors from calls to `E` will be from `try.go`. That's...probably OK? Nevertheless, it'd be better to be intentional about it, and either add line directives just before the panics below, or move F to its own file or the end of this file, so that line directive doesn't pollute the rest of the code.

Reply:
Test?

Reply:
I kept it in the same file, but moved it to the end.

PTAL.